aspnetcore/testing/razor-pages-testing/sample/tests/RazorPagesTestingSample.Tests/IntegrationTests/TestFixture.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Reflection;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.ViewComponents;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace RazorPagesTestingSample.Tests.IntegrationTests
+{
+    // A test fixture which hosts the target project (project we wish to test) in an in-memory server.
+    public class TestFixture<TStartup> : IDisposable
```

Comment:
@Eilon \ @danroth27  Do we want to create a work item to update this to use the new testing package once's it released?